docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1073 @@
+        ```
+        void GetCommands()
+        {
+            try
+            {
+                VoiceCommand current_command = new VoiceCommand();
+                VoiceCommandsGroup group = new VoiceCommandsGroup();
+
+                /// Other actions
+
+                current_commands = group.Commands;
+            }
+            catch (Exception e)
+            {
+                /// Error handling
+            }
+        }
+        ```
+
+4.  Register the commands for recognition by setting the command group to the voice control.
+
+    If you want to update registered commands, set the command group again with the updated commands.
+
+    ```
+    void SetCommand()
+    {
+        try
+        {
+            VoiceCommandsGroup group = new VoiceCommandsGroup();
+            VoiceCommand command = new VoiceCommand();
+            command.Command = "test";
+            command.Type = CommandType.Foreground;
+
+            group.Commands.Add(command);
+
+            VoiceControlManagerClient.SetCommands(group);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+5.  When no longer needed, unset the command group:
+
+    ```
+    /// Unset the command group
+    void UnsetCommand()
+    {
+        try
+        {
+            VoiceCommandsGroup group = new VoiceCommandsGroup();
+            VoiceCommand command = new VoiceCommand();
+            command.Command = "test";
+            command.Type = CommandType.Foreground;
+
+            group.Commands.Remove(command);
+
+            VoiceControlManagerClient.SetCommands(group);
+            ///or
+            VoiceControlManagerClient.ClearCommands();
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+
+<a name="other_commands"></a>
+## Other command functions
+
+- Register the commands for recognition by setting the command group to the voice control from file.
+    If you want to update registered commands, set the command group again with the updated commands.
+
+    ```
+    void SetCommandsFromFile(string commandFilePath)
+    {
+        try
+        {
+            /// file path contents.
+            /*
+                {
+                    "foreground": [
+                        {
+                        "format": "0",
+                        "domain": "0",
+                        "cmd": "open"
+                        },
+                        {
+                        "format": "0",
+                        "domain": "0",
+                        "cmd": "test"
+                        }
+                    ]
+                }
+            */
+
+            VoiceControlManagerClient.SetCommandsFromFile(commandFilePath, CommandType.Foreground);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+- Checks the command format is supported.
```

Comment:
@rishu-samsung I think you don't need to change the check to verify here.
The command format is may supported or not. 
So for the case of not supported command format, the error handling code is prepared. The developer will think another way for that case.

It is not a thing of "Verify". It is a thing to "Check".
So I hope you to focus on other more important things rather than changing "check" the "verify".